content/en/docs/tasks/observability/distributed-tracing/sampling/index.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Trace Sampling
+description: Learn the different approaches on how to configure trace sampling on the proxies.
+weight: 10
+keywords: [sampling,telemetry,tracing,opentelemetry]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+Istio provides multiple ways to configure trace sampling. In this page you will learn and understand
+all the different ways sampling can be configured.
+
+## Before you begin
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+
+## Available trace sampling configurations
+
+1.  Percentage Sampler: A random sampling rate for percentage of requests that will be selected for trace
+    generation.
+
+1.  Custom OpenTelemetry Sampler: A custom sampler implementation, that must be paired with the `OpenTelemetryTracingProvider`.
+
+1.  Deploy the OpenTelemetry Collector
+
+{{< boilerplate start-otel-collector-service >}}
```

Comment:
```suggestion
    {{< boilerplate start-otel-collector-service >}}
```

minor nit: these probably should be indented underneath the "Deploy" step above, otherwise the view looks like there's something missing and we go from a numbered list to a bullet list on a quick glance.

Otherwise this looks fine to me.

Reply:
Ah forgot about snippets in lists. Fixed!